prow/scheduler/reconciler.go

```diff
@@ -85,13 +89,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, fmt.Errorf("patch prowjob: %w", err)
 	}
 
-	logger.Info("Reconciliation completed")
+	log.Info("Reconciliation completed")
```

Comment:
is this log reallny needed? it holds little value

Reply:
Maybe should I remove `Reconciliation started` as well?

Reply:
yeah, I think so